index.html

```diff
@@ -255,17 +286,18 @@ <h2>
         the following steps:
       </p>
       <ol>
-        <li>If the <a>incumbent settings object</a> is not a <a>secure
+        <li>If the <a>current settings object</a> is not a <a>secure
```

Comment:
Things in the web platform generally use "relevant settings object of this `Navigator` object." We generally don't want to use "current". See https://html.spec.whatwg.org/multipage/webappapis.html#realms-settings-objects-global-objects, the example starting with "One reason why the relevant concept is generally a better default choice than the current concept"